resources/views/vendor/mail/html/button.blade.php

```diff
@@ -0,0 +1,19 @@
+<table class="action" align="center" width="100%" cellpadding="0" cellspacing="0" role="presentation">
```

Comment:
A simple button needs a whole table?

Reply:
Idk. These are pre-defiend by laravel (emails are pretty weird when it comes to responsivity)

Reply:
Hm. Do they look good on mobile too?